tests/unit/metadata/test_multi_table.py

```diff
@@ -3064,3 +3064,40 @@ def test_anonymize(self, mock_load):
             'parent_primary_key': 'col1',
             'child_foreign_key': 'col2',
         }
+
+    def test_update_columns_no_list_error(self):
+        # Setup
```

Comment:
minor: can we add small docstrings for both tests here adding a little more detail